Classes/Managers/LocalAssetsManager.swift

```diff
@@ -9,82 +9,168 @@
 import Foundation
 import AVFoundation
 
+// FairPlay is not available in simulators and is only downloadable in iOS10 and up.
+fileprivate let canDownloadFairPlay: Bool = {
+    if TARGET_OS_SIMULATOR==0, #available(iOS 10, *) {
+        return true
+    } else {
+        return false
+    }
+}()
+
+// Widevine is optional (and not available in simulators)
+fileprivate let canDownloadWidevineClassic: Bool = TARGET_OS_SIMULATOR==0 
+    && NSClassFromString("WidevineClassicAssetHandler") != nil
```

Comment:
@noamtamim  change it to `NSClassFromString("PlayKit.WidevineClassicAssetHandler")` , otherwise it returns nil